media-sound/skympc/skympc-1.6.4.recipe

```diff
@@ -5,15 +5,14 @@ variety of sound files while being controlled by its network protocol.
 
 SkyMPC a simple MPD client, powerd by Qt."
 HOMEPAGE="http://www.soramimi.jp/skympc/"
-COPYRIGHT="2015-2017 S.Fuchita"
+COPYRIGHT="2015-2018 S.Fuchita"
 LICENSE="GNU GPL v3"
 REVISION="1"
-srcGitRev="e24ba068a918634ecd269dda440c33362bb5dd51"
+srcGitRev="f6a96781319fb16b4d3a07c447d54ec476442a90"
 SOURCE_URI="https://github.com/soramimi/SkyMPC/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="7a05273938caed6a876e309346f4526eadf202223e7e08a177761cb6252193e5"
+CHECKSUM_SHA256="eb52ed5e834ff9f17c7d432143279c851bbf221ce97120df877678918dde743f"
 SOURCE_DIR="SkyMPC-$srcGitRev"
 SOURCE_FILENAME="$portVersionedName.tar.gz"
```

Comment:
```diff
-SOURCE_FILENAME="$portVersionedName.tar.gz"
+SOURCE_FILENAME="SkyMPC-$portVersion.tar.gz"
```
Using `$portVersionedName` in recipes that define `SECONDARY_ARCHITECTURES` is not recommended because `$portVersionedName` contains `$secondaryArchSuffix`, so on x86 secondary arch the archive would be renamed to `skympc_x86-1.6.4.tar.gz` (instead of `skympc-1.6.4.tar.gz`). While at it, you might wish to use `SkyMPC` instead of `skympc`.

BTW, you might also wish to remove `	cmd:git` from `BUILD_PREREQUIRES`.